contracts/toCKB-typescript/src/switch/liquidation_signertimeout/mod.rs

```diff
@@ -1,7 +1,52 @@
 use crate::switch::ToCKBCellDataTuple;
-use crate::utils::types::Error;
+use crate::utils::{
+    types::{ Error, ToCKBCellDataView},
+    tools::{ XChainKind, get_xchain_kind }
+};
 use core::result::Result;
+use ckb_std::{
+    ckb_constants::Source,
+    high_level::{load_cell_capacity, load_input_since},
+};
 
-pub fn verify(_toCKB_data_tuple: &ToCKBCellDataTuple) -> Result<(), Error> {
+pub fn verify(toCKB_data_tuple: &ToCKBCellDataTuple) -> Result<(), Error> {
+    let input_data = toCKB_data_tuple.0.as_ref().expect("inputs should contain toCKB cell");
+    let output_data = toCKB_data_tuple.1.as_ref().expect("outputs should contain toCKB cell");
+
+    // `get_toCKB_data_tuple` has checked cell nums == 1 in both inputs and outputs
+    // Todo: check since n4
+    let since = load_input_since(0, Source::GroupInput).expect("since should exist");
```

Comment:
move line 18 to the function `fn verify_since`

Reply:
fixed